DESCRIPTION

```diff
@@ -29,7 +29,7 @@ Roxygen: list(markdown = TRUE, r6 = TRUE)
 RoxygenNote: 7.3.2
 Depends: 
     R (>= 4.0)
-SystemRequirements: Cargo (Rust's package manager), rustc (Rust's compiler)
```

Comment:
This change is to be in accordance with CRAN's rust policy character by character